code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -474,6 +474,41 @@
 	spawn_tags = SPAWN_TAG_AMMO
 	rarity_value = 15
 
+/obj/item/ammo_casing/grenade/smoke
+	name = "FS SGR \"Conner\""
+	desc = "A standard-issue Frozen Star smoke screen round, for hiding troop movements."
+	icon_state = "grenade"
+	projectile_type = /obj/item/projectile/bullet/grenade/handgrenade/smoke
+	w_class = ITEM_SIZE_SMALL
+	caliber = CAL_GRENADE
+	is_caseless = TRUE
+	maxamount = 1
+	spawn_tags = SPAWN_TAG_AMMO
+	rarity_value = 15
+
+/obj/item/ammo_casing/grenade/flash
+	name = "FS FBR \"Stallone\""
+	desc = "A Duty-grade Frozen Star flash round, for clearing out criminal hideouts."
```

Comment:
What does "Duty-grade" mean here?

Reply:
leftover i can just change it to standard issue  like the rest